src/test_utils.cairo

```diff
@@ -0,0 +1,46 @@
+//! Test helpers
+
+/// Get bytes from hex (base16)
+pub fn from_hex(hex_string: ByteArray) -> ByteArray {
```

Comment:
shouldnt these 2 functions stay  in _utils.cairo_ ?  with unit test being either also in utils.cairo under a `tests` module, or in this new file?

Reply:
if i'm correct, `from_hex` is more than a util for test, its directly used in _validation.cairo_ file

Reply:
oh no ok its used in tests of validation.cairo, and to generate blocks which are also considered as tests.
you can ignore my comment

Reply:
Yeah, ideally it should be feature enabled for testing only